### PR TITLE
app-admin/logstash-bin: bump to 6.8.1/7.2.0

### PR DESCRIPTION
**Pull request by hydrapolic**
Thanks :)

### CODE REVIEW

File: app-admin/logstash-bin/logstash-bin-6.8.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+RESTRICT="strip"
+QA_PREBUILT="opt/logstash/vendor/jruby/lib/jni/*/libjffi*.so"
+
+RDEPEND="virtual/jre:1.8"
+
+S="${WORKDIR}/${MY_P}"
+
+pkg_setup() {
+	enewgroup ${MY_PN}
```
Comment (mgorny) [inline app-admin/logstash-bin/logstash-bin-6.8.1.ebuild]: Could you update it to use GLEP 81 user/group packages, and send RFC for UID/GID assignment? https://wiki.gentoo.org/wiki/Project:Quality_Assurance/UID_GID_Assignment is table of UIDs/GIDs already in use.

Comment (hydrapolic) [reply app-admin/logstash-bin/logstash-bin-6.8.1.ebuild]: Sure, will do.

File: acct-group/logstash/Manifest [hunk 2]
```diff
@@ -0,0 +1,2 @@
+EBUILD logstash-0.ebuild 201 BLAKE2B 3bd8012acd4f2fd399b140d0b4e0ee2f4d1465c5a140cff3296bcb5923c4abad28a4fc50502b55f04e1e8a5ccbceada9c1ae4de543c8f19f0484888feed3272f SHA512 5a3dfdfe0e17b7ca5723dd5027f7d3dc92669fd3ca138d8308ee29719a2711d0ddea985ebbba79d20a005482427f42be2199111fe2e1f7818ccb49afa9ef5730
```
Comment (mgorny) [inline acct-group/logstash/Manifest]: Not like this ;-P.

File: acct-group/logstash/logstash-0.ebuild [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="Logstash program group"
+ACCT_GROUP_ID=270
+KEYWORDS="~amd64"
```
Comment (mgorny) [inline acct-group/logstash/logstash-0.ebuild]: No KEYWORDS.

File: app-admin/logstash-bin/logstash-bin-6.8.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+# source: LICENSE.txt and NOTICE.txt
+LICENSE="Apache-2.0 MIT x-pack? ( Elastic )"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="x-pack"
+
+RESTRICT="strip"
+QA_PREBUILT="opt/logstash/vendor/jruby/lib/jni/*/libjffi*.so"
+
+
```
Comment (mgorny) [inline app-admin/logstash-bin/logstash-bin-6.8.1.ebuild]: DoubleEmptyLine™

File: acct-user/logstash/logstash-0.ebuild [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+inherit acct-user
+
+DESCRIPTION="Logstash program user"
+ACCT_USER_ID=270
+ACCT_USER_HOME=/var/lib/logstash
+ACCT_USER_HOME_PERMS=0770
+ACCT_USER_GROUPS=( logstash )
+acct-user_add_deps
+KEYWORDS="~amd64"
```
Comment (mgorny) [inline acct-user/logstash/logstash-0.ebuild]: Same.

File: acct-user/logstash/logstash-0.ebuild [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
```
Comment (mgorny) [inline acct-user/logstash/logstash-0.ebuild]: This is out of order.

File: acct-group/logstash/logstash-0.ebuild (hunk 3 above)
Comment (hydrapolic) [reply acct-group/logstash/logstash-0.ebuild]: I saw it in mysql, removing.

File: acct-user/logstash/logstash-0.ebuild (hunk 6 above)
Comment (hydrapolic) [reply acct-user/logstash/logstash-0.ebuild]: You mean the commit? It's strange, because locally this commit is the second one, hm..

Comment (hydrapolic) [reply acct-user/logstash/logstash-0.ebuild]: After another rebase and force push it seems fine now.

File: app-admin/logstash-bin/logstash-bin-6.8.1.ebuild (hunk 4 above)
Comment (hydrapolic) [reply app-admin/logstash-bin/logstash-bin-6.8.1.ebuild]: Removed :)

**Comment by hydrapolic** [comment]
@mgorny  I thought only devs can send to gentoo-dev. If you say it's ok I'll send a notice to gentoo-dev about the UID/GID 270 allocation.

**Comment by hydrapolic** [comment]
I haven't found logstash user/gid in other databases (fedora, etc.)

**Comment by juippis** [comment]
Every proxied maintainer was whitelisted (allowed) to post on gentoo-dev, because many proxy-maint tasks require that right. Unless you're explicitly banned... go for it :)

**Comment by hydrapolic** [comment]
https://archives.gentoo.org/gentoo-dev/message/ee2cb65a29eb671bd2f6d1730ebc1883

**Comment by hydrapolic** [comment]
TODO:
drop ACCT_USER_HOME_PERMS=0770
drop keepdir "/var/lib/${MY_PN}" from the ebuild

**Comment by hydrapolic** [comment]
Updated per https://archives.gentoo.org/gentoo-dev/message/63d07af38f5fc37b9d84f19c7d6f2bd4

The perms on the home dir are set via the init script to 0750, so setting in the acct-user/logstash ebuild too. Removed duplicate keepdir from logstash ebuilds.

**Comment by hydrapolic** [comment]
Thanks @mgorny @juippis

**Pull request closed without merging**